llvm/test/CodeGen/AMDGPU/sdwa-peephole-multiuse-kill.ll

```diff
@@ -0,0 +1,194 @@
+; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
```

Comment:
Better to just check the final ISA? It would be better to have a supplemental MIR test that has an explicit kill flag, a lot of things could have lost the flag before this point

Reply:
I've just replaced with a reduced MIR test. Without the flag clearing, PeepholeSDWA will attach kill flag to a folded reg, which it will subsequently violate.